samples/drawing/drawing.cpp

```diff
@@ -1896,15 +1887,18 @@ void MyCanvas::Draw(wxDC& pdc)
     }
 
     if ( m_clip )
-        dc.SetClippingRegion(100, 100, 100, 100);
+    {
+        int clipSize = dc.FromDIP(100);
```

Comment:
Sorry for nitpicking, but this is not a sizer, is it? I.e. it's used also as `x` and `y` arguments.